### DISCUSSION TITLE
Dependent on Microsoft.Xaml.Behaviors or not?

### DISCUSSION THREAD

**Discussion by znakeeye** [PrismLibrary/Prism#2652] [WPF]
`Prism.Wpf` depends on `Microsoft.Xaml.Behaviors`. However, when we run our prism-powered application this DLL is never loaded. Exactly what depends on this library?

I'm trying to figure out when to include the DLL in our deployed package. So far, it runs without it - even though `Prism.Wpf.dll` indeed has a referencet o `Microsoft.Xaml.Behaviors.dll`. Some kind of delay loading?

**Comment by brianlagunas** [comment]
The Prism Behaviors depend on it.

**Reply by znakeeye** [reply]
Ok, so as long as we don't instantiate any of those there is no need to deploy the DLL?

(I take it there is no strong reference to this dll. Is it loaded via `Assembly.Load()`?)
